performance_regression_gradual_grow_throughput.py

```diff
@@ -170,13 +183,15 @@ def check_latency_during_steps(self, step):
             self.update({"latency_during_ops": latency_results})
             return latency_results
 
-    def run_step(self, stress_cmds, current_throttle, num_threads):
+    def run_step(self, stress_cmds, current_throttle, num_threads, step_duration):
         results = []
         stress_queue = []
         for stress_cmd in stress_cmds:
             params = {"round_robin": True, "stats_aggregate_cmds": False}
             stress_cmd_to_run = stress_cmd.replace(
                 "$threads", f"{num_threads}").replace("$throttle", f"{current_throttle}")
+            if step_duration is not None:
```

Comment:
The condition must be opposite - if it is None then we fail right away because the cmd anyway will be incorrect.

Reply:
For write load the duration is not relevant - this is the reason why I check for `not None`
```
perf_gradual_step_duration: {"read": '20m', "write": None, "mixed": '20m'}
```
Which means it's right, and just like we thought it would be